lib/src/chain.dart

```diff
@@ -102,7 +102,7 @@ class Chain implements StackTrace {
     return runZoned(() {
       try {
         return callback();
-      } catch (error, stackTrace) {
+      } on Object catch (error, stackTrace) {
```

Comment:
wha?

Reply:
The commit message calls this out - it won't be necessary at some point.

https://github.com/dart-lang/language/issues/922